supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -18,6 +18,16 @@
 
 *See also*:
 
+[[ibm-eserver-system-i]]
+==== image:images/yes.png[no] IBM eServer System i (noun)
```

Comment:
If you want this to be a use it: no, then it has to be "no.png" here and the "use it" field has to be set to "no".

Reply:
Well spotted, and thanks for reviewing. Updated.